notebooks/__init__.py

```diff
@@ -0,0 +1,91 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Functionality for computing metrics, with a particular focus on group metrics.
```

Comment:
I'm not sure I understand the purpose of this file. This is all functionality from fairlearn.metrics, isn't it? Why not directly import into your notebook?